styles.css

```diff
@@ -0,0 +1,733 @@
+:root{
+    --primary-color:  #374151 ;
+    --dark-pink: #e70161;
+    --border-color: rgb(146, 146, 146);
+
+}
+body
+{
+    margin: 0rem;
+    padding: 0rem;
+}
+.a-nav2
+{
+    text-decoration: none;
+    color: var(--dark-pink);
+}
+.navigation
+{
+    background-color: var(--primary-color);
+    color:rgb(250, 250, 255);
+    height: 4rem;
+    border-radius: 1rem;
+}
+.container
+{
+    padding: 0rem 1rem;
+}
+.navigation .nav-developer
+{
+font-weight: bold;
+padding: 1rem;
+font-size: 25px;
+display: inline;
+}
+.list-non-bullet
+{
+    list-style: none;
+    padding-inline-start: 800px;
+   padding-block-start: 1rem;
+    font-size: 22px;
```

Comment:
It's good to follow a single sizing unit inside a class or an attribute. Also, using using `rem` makes things a bit responsive.